src/main/java/cn/nukkit/network/process/processor/SettingsCommandProcessor.java

```diff
@@ -0,0 +1,26 @@
+package cn.nukkit.network.process.processor;
+
+import cn.nukkit.Player;
+import cn.nukkit.PlayerHandle;
+import cn.nukkit.network.process.DataPacketProcessor;
+import cn.nukkit.network.protocol.ProtocolInfo;
+import cn.nukkit.network.protocol.SettingsCommandPacket;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Locale;
+
+public class SettingsCommandProcessor extends DataPacketProcessor<SettingsCommandPacket> {
+
+    @Override
+    public void handle(@NotNull PlayerHandle playerHandle, @NotNull SettingsCommandPacket pk) {
+        Player player = playerHandle.player.asPlayer();
+
+        String command = pk.command.toLowerCase(Locale.ENGLISH);
+        player.getServer().executeCommand(player, command);
```

Comment:
I don't remember, Server#executeCommand() has checks for permissions or not?

Reply:
I'll take a look at it

Reply:
> I don't remember, Server#executeCommand() has checks for permissions or not?

good